packages/components/src/border-box-control/border-box-control-split-controls/component.tsx

```diff
@@ -36,16 +36,25 @@ const BorderBoxControlSplitControls = (
 		__next36pxDefaultSize,
 		...otherProps
 	} = useBorderBoxControlSplitControls( props );
-	const containerRef = useRef();
-	const mergedRef = useMergeRefs( [ containerRef, forwardedRef ] );
-	const popoverProps = popoverPlacement
-		? {
+	const [ popoverProps, setPopoverProps ] = useState< PopoverPartialProps >();
+	const [ popoverAnchor, setPopoverAnchor ] = useState< Element >();
+
+	const containerRef = useCallback( ( node ) => {
+		setPopoverAnchor( node );
+	}, [] );
+
+	useEffect( () => {
+		if ( popoverPlacement ) {
```

Comment:
Final note: I'm not in love with the choice of adding `popoverPlacement` and `popoverOffset`, but these were added as part of #40740 and it's not in the scope of this PR to revisit the APIs of `BorderBoxControl`